src/types/products.ts

```diff
@@ -2,14 +2,26 @@ import { ProductMeta, Products } from '@hashicorp/platform-product-meta'
 import { NavigationHeaderItem } from 'components/navigation-header/types'
 import { MenuItem } from 'components/sidebar'
 
+type ProductName =
+  | 'Boundary'
+  | 'Consul'
+  | 'HashiCorp Cloud Platform'
+  | 'Nomad'
+  | 'Packer'
+  | 'Sentinel'
+  | 'Terraform'
+  | 'Vagrant'
+  | 'Vault'
+  | 'Waypoint'
```

Comment:
Added this type because we know what these values are and we also have all of these products in DevDot (at varying levels).